projects/systelab-translate/package.json

```diff
@@ -19,10 +19,10 @@
   },
   "private": false,
   "peerDependencies": {
-    "@angular/common": "^20.0.0",
-    "@angular/core": "^20.0.0",
-    "@ngx-translate/core": "^16.0.0",
-    "@ngx-translate/http-loader": "^16.0.0",
-    "date-fns": "^2.30.0"
+    "@angular/common": "20.0.0",
```

Comment:
In this case, if we remove the caret we're having issues when inslling the library because we're using Angular 20.1.1 and here we're fixing the dependency for 20.0.0